Exercises/4-count-types.js

```diff
@@ -1,5 +1,26 @@
 'use strict';
 
-const countTypesInArray = null;
+// const testArray = [
+//   true,
+//   1,
+//   -1,
+//   0,
+//   'Test',
+//   null,
+//   undefined,
+//   -Infinity,
+//   Infinity,
+//   [1, 2, 3],
+//   { obj: 'obj' },
+// ];
+
+const countTypesInArray = input => {
+  const r = {};
+  for (const el of input) {
+    const tor = typeof el;
+    r[tor] = (r[tor]) ? r[tor] + 1 : 1;
```

Comment:
```suggestion
    r[tor] = r[tor] ? r[tor] + 1 : 1;
```

Reply:
But better put previous counter to variable `const count = r[tor];` then increment and put to collection

Reply:
Improve naming: `r` is very short name, `tor` - is not human friendly readable.